src/main/java/uk/gov/digital/ho/hocs/audit/service/CaseDataExportService.java

```diff
@@ -143,7 +143,7 @@ protected ExportDataConverter getDataConverter(boolean convert, CaseTypeDto case
         uuidToName.putAll(
             infoClient.getUnits().stream().collect(Collectors.toMap(UnitDto::getUuid, UnitDto::getDisplayName)));
         caseworkClient.getAllCaseTopics().forEach(
-            topic -> uuidToName.putIfAbsent(topic.getTopicUUID().toString(), topic.getTopicText()));
+            topic -> uuidToName.putIfAbsent(topic.getUuid().toString(), topic.getTopicText()));
```

Comment:
We should keep mapping the topicUUID as well in case any events are expecting that mapping.
```suggestion
            topic -> {
                uuidToName.putIfAbsent(topic.getTopicUUID().toString(), topic.getTopicText());
                uuidToName.putIfAbsent(topic.getUuid().toString(), topic.getTopicText());
            }
        );
```